docs/products/kafka/kafka-connect/howto/clickhouse-sink-connector.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Create a ClickHouse sink connector for Aiven for Apache Kafka®
+sidebar_label: ClickHouse sink connector
+---
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import ConsoleLabel from "@site/src/components/ConsoleIcons";
+
+The ClickHouse sink connector delivers data from Apache Kafka® topics to a ClickHouse database for efficient querying and analysis.
+
+## Prerequisites
+
+Before you begin, ensure that you have the following:
+
+- An [Aiven for Apache Kafka® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/howto/enable-connect)
+  with Apache Kafka Connect enabled or a
+  [dedicated Aiven for Apache Kafka Connect® service](https://docs.aiven.io/docs/products/kafka/kafka-connect/get-started#apache_kafka_connect_dedicated_cluster).
+- Access to a ClickHouse service (either Aiven for ClickHouse or an external instance),
+  including:
+  - Hostname, port, and credentials for the ClickHouse service.
+  - A pre-created target database and table.
+
+## Limitations
+
+The ClickHouse sink connector has the following limitations related to data consistency
+and exactly-once delivery:
+
+1. **No exactly-once delivery after restore**: The connector does not guarantee
+   exactly-once delivery after a restore, power-off, or service forking. This may result
+   in duplicate records in ClickHouse.
+
+1. **Manual removal of duplicate records**: If duplicates occur, manually remove them
+   to maintain data consistency in ClickHouse.
+
+   Ensure that all potential duplicates are processed before removal:
```

Comment:
Might be an idea to put the instructions in their own section and link to it instead of putting it under the limitations section.